utilities.ts

```diff
@@ -971,6 +971,34 @@ module J2ME {
   }
 
   export module HashUtilities {
+    // https://github.com/garycourt/murmurhash-js
```

Comment:
I think we'll need to do something a bit different here to reference the license. I'll ping my license guy.